src/js/components/RadioButton/doc.js

```diff
@@ -116,6 +116,12 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '24px',
   },
+  'radioButton.hover.background.color': {
+    description: `The color of the Box surronding the RadioButton
```

Comment:
```suggestion
    description: `The background color of the Box surrounding the RadioButton
```

Reply:
Think you'll have to pull this down locally to update the snapshots properly.

Reply:
yes I believe so then Ill have to run yarn update